### ISSUE TITLE
Return keyboard focus to form errors in signup form

### ISSUE THREAD

**Issue by emmastephenson** [CDCgov/prime-simplereport#4015]
## Area or Component
Sign up form (organization details and personal details)

## Quick Summary
The form doesn't return keyboard focus to the offending element in case of an erroring submission. Instead it scrolls to the top of the page. It should scroll to the first error (ideally after focusing the keyboard on the toast, but that's a separate problem)

### How to Replicate
Trigger an error on form submission

## Accessibility Violation
https://dequeuniversity.com/class/forms/form-validation/focus-first-error

## Action Requested
Send focus to the first error on form submission.

If not already the case, we should also make sure to:
- Set invalid fields to aria-invalid="true".
- Ensure error messages are associated with form fields using aria-describedby so that screen reader users know how to fix the problem.
